sentry/src/main/java/io/sentry/clientreport/DiscardReason.java

```diff
@@ -0,0 +1,23 @@
+package io.sentry.clientreport;
+
+public enum DiscardReason {
+  QUEUE_OVERFLOW("queue_overflow"),
+  CACHE_OVERFLOW("cache_overflow"),
+  RATELIMIT_BACKOFF("ratelimit_backoff"),
+  NETWORK_ERROR("network_error"),
+  SAMPLE_RATE("sample_rate"),
+  BEFORE_SEND("before_send"),
+  EVENT_PROCESSOR("event_processor"), // also for ignored exceptions
+  STORAGE_ERROR("storage_error"), // TODO MOBILE-207 what should we use?
```

Comment:
What is `MOBILE-207`?

Reply:
enum value should be removed.

`MOBILE-207` is the Jira ticket. I use this to more easily find the TODOs of the current task or when leaving a TODO for a future task.

Reply:
Guess I should use github issue numbers instead of JIRA ticket numbers.

Reply:
Yeah, will do from now on